cmd/ar+https/main.go

```diff
@@ -17,12 +17,17 @@ package main
 import (
 	"bufio"
 	"context"
+	"fmt"
 	"os"
 
 	"github.com/GoogleCloudPlatform/artifact-registry-apt-transport/apt"
 )
 
 func main() {
 	apt := apt.NewAptMethod(bufio.NewReader(os.Stdin), os.Stdout)
-	apt.Run(context.Background())
+	err := apt.Run(context.Background())
+	if err != nil {
+		fmt.Fprintf(os.Stderr, "%+v\n", err)
+		os.Exit(100)
```

Comment:
why 100?

Reply:
This was addressed [in the review for pull request #21](https://github.com/GoogleCloudPlatform/artifact-registry-apt-transport/pull/21#discussion_r1548322842): 100 is the canonical error code returned by apt plugins when something goes wrong.